index.html

```diff
@@ -1,79 +1,226 @@
+            </div>
+         </section>
+         </div>
+         <section id="works">
+          <div class="card_works">
+            <img class="card_works_image" src="./assets/tonic_1.svg" alt="image">
+            <h3 class="card_works_heading">Tonic</h3>
+            <div class="card_works_links">
+              <span>Canopy</span>
+              &middot;
+              <span>Backend Dev</span>               
+              &middot;
+              <span>2015</span>
+           </div>
+             <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+             <div class="card_works_tags">
+               <span>html</span>
+               <span>css</span>
+               <span>javascript</span>
+             </div>
+             <button class="card_works_button">See Project</button>
           </div>
-          <div class="humberger menu"></div>
-          <div class="humberger">
-            <img src="humberger.svg" alt="humberger icon" />
+          <div class="card_works">
+           <img class="card_works_image" src="./assets/multi_post_1.svg" alt="image">
+           <h2 class="card_works_heading">Multi-Post Stories</h2>
+           <div class="card_works_links">
+             <span>Canopy</span>
+             &middot;
+             <span>Backend Dev</span>               
+             &middot;
+             <span>2015</span>
           </div>
-        </nav>
-      </header>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="card_works_tags">
+              <span>html</span>
+              <span>css</span>
+              <span>javascript</span>
+            </div>
+            <button class="card_works_button">See Project</button>
+         </div>
+         <div class="card_works">
+           <img class="card_works_image" src="./assets/tonic_2.svg" alt="image">
+           <h2 class="card_works_heading">Tonic</h2>
+           <div class="card_works_links">
+             <span>Canopy</span>
+             &middot;
+             <span>Backend Dev</span>               
+             &middot;
+             <span>2015</span>
+          </div>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="card_works_tags">
+              <span>html</span>
+              <span>css</span>
+              <span>javascript</span>
+            </div>
+            <button class="card_works_button">See Project</button>
+         </div>
+         <div class="card_works">
+           <img class="card_works_image" src="./assets/multi_post_2.svg" alt="image">
+           <h2 class="card_works_heading">Multi-Post Stories</h2>
+           <div class="card_works_links">
+             <span>Canopy</span>
+             &middot;
+             <span>Backend Dev</span>               
+             &middot;
+             <span>2015</span>
+          </div>
```

Comment:
Please ensure that you stick to the design as much as possible (e.g., font, colors, images, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). Please ensure that you this adjustment to all the work cards.

Your Implementation | Figma Design
:---:|:---:
![image](https://user-images.githubusercontent.com/10332499/154817104-2162029f-cc20-4d48-94aa-9cf6a2cadea6.png) | ![image](https://user-images.githubusercontent.com/10332499/154817163-54f12f0e-b3c7-41f9-bcb3-3212a0b04fae.png)